File: coding/14th August 2025/Largest 3-Same-Digit Number in String.java
//Largest 3-Same-Digit Number in String
class Solution 
{
    public String largestGoodInteger(String num) 
    {
        String[] arr = {"999", "888", "777", "666", "555", "444", "333", "222", "111", "000"};
        for(String str : arr)
            if(num.contains(str))
                return str;
        return "";
    }
}

File: Coding in Python/16th March 2026/Largest 3-Same-Digit Number in String.py
#Largest 3-Same-Digit Number in String
class Solution:
    def largestGoodInteger(self, num: str) -> str:
        arr = ["999", "888", "777", "666", "555", "444", "333", "222", "111", "000"]
        for ele in arr:
            if ele in num:
                return ele
        return ""
